.vuestorefrontcloud/docker/default.conf

```diff
@@ -0,0 +1,13 @@
+server {
```

Comment:
Would it be possible to have this file in a separate directory `nginx`? I think it would be easier to manage later if we know from the very beginning that this file is related to Nginx rather that to Docker itsefl ;)

Reply:
I think it is unnecessary, there is only a one docker image, and it is nginx

Reply:
Ok, so let's leave it like that